src/helper/class-ee-site.php

```diff
@@ -102,9 +102,9 @@ public function _list( $args, $assoc_args ) {
 		$sites = Site::all();
 
 		if ( $enabled && ! $disabled ) {
-			$sites = Site::where( 'site_enabled', true );
+			$sites = Site::where( 'site_enabled', 1 );
```

Comment:
I think here `true/false` is makes more sense. So you can maybe update it in where function. Like if you know that we are not storing the value in sqlite with true/false and it's 1/0 then you can do this in where function.

```
if ( is_bool( $value ) ) {
   $value =  (int) $value;
} 
```

Reply:
Yes, that will be better.